app/src/main/res/values/strings.xml

```diff
@@ -38,9 +38,9 @@
     <string name="title_activity_calibration_override">Calibration Override</string>
     <string name="pref_header_cloud_storage">Cloud Storage</string>
     <string name="auto_configure_title">Auto configure</string>
-    <string name="prefs_auto_config_summary">Auto configure using a barcode.</string>
-    <string name="scan_share2_barcode">Scan Share Barcode</string>
-    <string name="pref_share2_scan_barcode_summary">Or scan the barcode on the share receiver</string>
+    <string name="prefs_auto_config_summary">Auto configure using a QR code.</string>
+    <string name="scan_share2_barcode">Scan Share Receiver barcode</string>
```

Comment:
In that case, we should remove this feature completely from xDrip. This will increase maintainability and also avoids work for translators of new languages like Estonian.